be/src/olap/rowset/segment_v2/column_writer.cpp

```diff
@@ -262,6 +262,11 @@ Status ColumnWriter::create(const ColumnWriterOptions& opts, const TabletColumn*
         }
         case FieldType::OLAP_FIELD_TYPE_MAP: {
             DCHECK(column->get_subtype_count() == 2);
+            if (column->get_subtype_count() < 2) {
+                return Status::InvalidArgument(
+                        "If you upgraded from version 1.2.*, please drop the "
+                        "map column and wait a while to re-create the map column.");
```

Comment:
"drop the MAP column and wait a while to re-create the map column" => "DROP the MAP columns and then ADD the MAP columns back."